crates/pathfinder/src/storage/state.rs

```diff
@@ -1471,13 +1488,14 @@ mod tests {
 
             mod latest {
                 use super::*;
+                use crate::storage::test_utils;
 
                 #[test]
                 fn some() {
                     let storage = Storage::in_memory().unwrap();
                     let connection = storage.connection().unwrap();
 
-                    let blocks = create_blocks();
+                    let blocks = test_utils::create_blocks();
```

Comment:
This is probably a conflict resolution aftermath with both changes being present. I think the long form with `test_utils::create_blocks` would be preferable because it's longer form :)

Reply:
I'm currently cleaning the history of some commented code and conflict left-overs right now and already locally I cannot see any instance of `create_blocks` being used without the explicit `test_utils` qualifier in tests.